eel/tests/p2p_connection_test.rs

```diff
@@ -31,23 +33,26 @@ mod p2p_connection_test {
     #[file_serial(key, "/tmp/3l-int-tests-lock")]
     fn test_p2p_connection_with_unreliable_lsp() {
         nigiri::ensure_environment_running();
+
+        // Start the node when lspd isn't available
+        nigiri::pause_lspd();
         let node = mocked_storage_node().start_or_panic();
 
-        // Test disconnect when LSP is down.
-        {
-            // Let's shutdown LSPD LND.
-            nigiri::pause_lspd();
-            wait_for_eq!(node.get_node_info().num_peers, 0);
-        }
+        assert_eq!(
+            node.query_lsp_fee(),
+            Err(runtime_error(
+                RuntimeErrorCode::LspServiceUnavailable,
+                "Failed to get LSP info"
+            ))
+        );
+        wait_for_eq!(node.get_node_info().num_peers, 0);
```

Comment:
Why do you wait for that? There shouldn't have been any peer since the node was started, right?

Reply:
Yes, it something that remained from the previous code. Fixed!